### DISCUSSION TITLE
Kata 10, errir 13

### DISCUSSION THREAD

**Discussion by MauricioNovelo** [LaunchX-InnovaccionVirtual/CursoIntroPython#261] [Q&A]
Tengo una duda con este ejercicio en donde me debería dar el error 21, pero me arroja el error 13
![image](https://user-images.githubusercontent.com/83683322/153986542-986f554e-144e-4391-a4f3-739770a9c716.png)

**Comment by doguedogue** [comment]
checa https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/214

**Reply by MauricioNovelo** [reply]
Bien gracias, le puse una terminal basada en linux y ya da el error esperado
